### PR TITLE
Add map

### PR DESCRIPTION
**Pull request by tombein**

### CODE REVIEW

File: src/components/Cards/Cards.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import Offercard from '../styled-components/StyledOfferCard';
+import HeadlineOffercard from '../styled-components/styled-headline-offercard';
```
Comment (FranziGraml) [inline src/components/Cards/Cards.js]: Du könntest deine Styles-Ordner auch so benennen.... OfferCard.styles.js sowie HeadlineOffercard.styles.js
Dann wäre es einheitlicher :-)

**Review by FranziGraml — APPROVED** [review]
Hi Tom,
nice work :-)

File: src/components/Cards/Cards.js (hunk 1 above)
Comment (doemser) [reply src/components/Cards/Cards.js]: good point, you should definitely go for a uniform naming.

**Review by doemser — CHANGES_REQUESTED** [review]
Hey, looks like you did good work there ;)

You should give your naming a little revision, CardMap and CardDaten are pretty unusual namings, which are not self-explanatory for someone who reviews you code. Try to make it more easier to understand for people who dont know what your project is about.

**Review by doemser — APPROVED** [review]
Looks good now :)

**Pull request merged**